source/patterns/@aws-solutions-constructs/aws-route53-apigateway/lib/index.ts

```diff
@@ -107,26 +112,22 @@ export class Route53ToApiGateway extends Construct {
         throw new Error('Public APIs require an existingHostedZone be passed in the Props object.');
       } else {
         if (!props.privateHostedZoneProps) {
-          throw new Error('Must supply privateHostedZoneProps to create a private API');
+          throw new Error('Must supply privateHostedZoneProps to create a private API.');
```

Comment:
This message should be 

'Must provide either existingHostedZoneInterface or privateHostedZoneProps.'

Reply:
Line 108 above should be:

'Must provide either existingHostedZoneInterface or privateHostedZoneProps, but not both.'